### ISSUE TITLE
"Something went wrong" issue on Jobs Settings by trying to edit [worker-json.js not found]

### ISSUE THREAD

**Issue by prinzpi** [ansible/awx#11922]
**ISSUE TYPE**
- Bug Report

**Components**
- UI

**SUMMARY**
By trying to edit the Jobs Settings (Settings -> Jobs Settings) I'm receiving the error message "Something went wrong".
![image](https://user-images.githubusercontent.com/12089703/158562848-6ff85700-e9bb-4a30-8dae-0ab5743a891a.png)

The Details also attached at the bottom. Anybody an idea what's happening here? By editing the other things inside the "Settings" area I do not receive any errors.

**ENVIRONMENT**
- AWX version: 0.18.0
- kubernetes version: 1.25.2
- AWX install method: minikube
- Used "service_type": LoadBalancer

**Details**
`gY@http://192.168.49.2:32567/static/js/main.ac5f89fb.js:2:2990670 oo@http://192.168.49.2:32567/static/js/main.ac5f89fb.js:2:1270463 Ks@http://192.168.49.2:32567/static/js/main.ac5f89fb.js:2:1322763 Ol@http://192.168.49.2:32567/static/js/main.ac5f89fb.js:2:1310006 Tl@http://192.168.49.2:32567/static/js/main.ac5f89fb.js:2:1309934 El@http://192.168.49.2:32567/static/js/main.ac5f89fb.js:2:1309795 yl@http://192.168.49.2:32567/static/js/main.ac5f89fb.js:2:1306761 34463/Zi/<@http://192.168.49.2:32567/static/js/main.ac5f89fb.js:2:1256255 36813/t.unstable_runWithPriority@http://192.168.49.2:32567/static/js/main.ac5f89fb.js:2:1482480 Gi@http://192.168.49.2:32567/static/js/main.ac5f89fb.js:2:1256032 Zi@http://192.168.49.2:32567/static/js/main.ac5f89fb.js:2:1256202 Ki@http://192.168.49.2:32567/static/js/main.ac5f89fb.js:2:1256135 pl@http://192.168.49.2:32567/static/js/main.ac5f89fb.js:2:1304138 Io@http://192.168.49.2:32567/static/js/main.ac5f89fb.js:2:1275790 la/e/<@http://192.168.49.2:32567/static/js/main.ac5f89fb.js:2:1633449 u@http://192.168.49.2:32567/static/js/main.ac5f89fb.js:2:1343532 19727/c/a._invoke@http://192.168.49.2:32567/static/js/main.ac5f89fb.js:2:1343321 19727/k/</<@http://192.168.49.2:32567/static/js/main.ac5f89fb.js:2:1343961 i@http://192.168.49.2:32567/static/js/main.ac5f89fb.js:2:1539957 s@http://192.168.49.2:32567/static/js/main.ac5f89fb.js:2:1540161`

### Update:

The above screenshot is within Firefox Browser. I have installed chrome to check if it's maybe related to a browser issue. But with Google Chrome the same error is appearing.

![image](https://user-images.githubusercontent.com/12089703/158621157-813d239c-9602-4f9e-a911-b1c4ed62a7a5.png)

I have used the chrome inspector and it seems some json worker / files can't be found?!

```
Failed to load resource: the server responded with a status of 404 (Not Found)
:32567/worker-json.js:1          Failed to load resource: the server responded with a status of 404 (Not Found)
:32567/worker-json.js:1          Failed to load resource: the server responded with a status of 404 (Not Found)
:32567/worker-json.js:1          Failed to load resource: the server responded with a status of 404 (Not Found)
react-dom.production.min.js:216 TypeError: Cannot read properties of undefined (reading 'default')
    at gY (SharedFields.js:444:54)
    at oo (react-dom.production.min.js:157:137)
    at Ks (react-dom.production.min.js:267:460)
    at Ol (react-dom.production.min.js:250:347)
    at Tl (react-dom.production.min.js:250:278)
    at El (react-dom.production.min.js:250:138)
    at yl (react-dom.production.min.js:243:163)
    at react-dom.production.min.js:123:115
    at t.unstable_runWithPriority (scheduler.production.min.js:18:343)
    at Gi (react-dom.production.min.js:122:325)
us @ react-dom.production.min.js:216
worker-json.js:1          GET http://192.168.49.2:32567/worker-json.js 404 (Not Found)
worker-json.js:1          GET http://192.168.49.2:32567/worker-json.js 404 (Not Found)
worker-json.js:1          GET http://192.168.49.2:32567/worker-json.js 404 (Not Found)
worker-json.js:1          GET http://192.168.49.2:32567/worker-json.js 404 (Not Found)
react-dom.production.min.js:216 TypeError: Cannot read properties of undefined (reading 'default')
    at gY (SharedFields.js:444:54)
    at oo (react-dom.production.min.js:157:137)
    at Ks (react-dom.production.min.js:267:460)
    at Ol (react-dom.production.min.js:250:347)
    at Tl (react-dom.production.min.js:250:278)
    at El (react-dom.production.min.js:250:138)
    at yl (react-dom.production.min.js:243:163)
    at react-dom.production.min.js:123:115
    at t.unstable_runWithPriority (scheduler.production.min.js:18:343)
    at Gi (react-dom.production.min.js:122:325)
```

**Comment by wiglaf1988** [comment]
same thing here in the job settings page,thanks for checking

TypeError: Cannot read properties of undefined (reading 'default') at gY (https://url/static/js/main.ac5f89fb.js:2:2990705) at oo (https://url/static/js/main.ac5f89fb.js:2:1270463) at Ks (https://url/static/js/main.ac5f89fb.js:2:1322763) at Ol (https://url/static/js/main.ac5f89fb.js:2:1310004) at Tl (https://url/static/js/main.ac5f89fb.js:2:1309932) at El (https://url/static/js/main.ac5f89fb.js:2:1309795) at yl (https://url/static/js/main.ac5f89fb.js:2:1306761) at https://url/static/js/main.ac5f89fb.js:2:1256255 at t.unstable_runWithPriority (https://url/static/js/main.ac5f89fb.js:2:1482480) at Gi (https://url/static/js/main.ac5f89fb.js:2:1256032)

**Comment by lutskevich** [comment]
same issues
awx-operator version: 0.18.0
AWX 20.0.1

**Comment by zylpsrs** [comment]
same issues
awx-operator version: 0.18.0
AWX 20.0.1

**Comment by kurokobo** [comment]
This appears to be solved via #11896.

**Comment by nixocio** [comment]
Thanks @kurokobo, this issue was indeed addressed by https://github.com/ansible/awx/pull/11896
